src/assets/css/styles.head.css

```diff
@@ -1,5 +1,5 @@
 body {
-  background-color: #013165;
+  background: #edf0f5;
```

Comment:
can you please use constants

Reply:
The `styles.head.css` file wasn't using variables, so I kept like it was, using absolute values, and checking the webpack.config I think it isn't processed through postcss